views/about.twig

```diff
@@ -1,14 +1,181 @@
+
+            <h3><a name="after-talks">After The Talks</a></h3>
+
+            <p>When the talks are finished, usually around 9pm, most of the group head over to Camden. This part of the meeting is your opportunity to socialise with other PHPDorset members and speak with the presenters about their talks.</p>
+
+            <p>Just because we’re in a pub doesn’t mean you have to drink alcohol. If you do choose to drink alcohol we encourage you to explore the food menu and drink responsibly.</p>
+
+            <p>Although the venue for this part of the meeting is a pub and therefore feels more informal, be aware that the code of conduct still applies.</p>
+
+            <h3><a name="sponsors">Sponsorship</a></h3>
+
+            <p>PHPDorset is sponsored by <a href="http://www.spectrumit.co.uk/">SpectrumIT</a> who cover the venue and pizzas. We don’t currently need venue sponsorship or an alternate venue or main sponsor.</p>
+
+            <p>There are opportunities for drinks sponsorship during the meeting, or bar-tab sponsorship after the meeting at the pub. If you would like to provide sponsorship you should get in touch with us (@phpdorset, phpdorset@gmail.com) to find out the prices and arrange it.</p>
+
+            <p>In return for your sponsorship PHPDorset will:
+                <ul>
+                    <li>put your logo in the sponsor side-bar on the relevant meeting page on phpdorset.co.uk</li>
+                    <li>mention the specifics of your sponsorship in the main details of the relevant meeting page on phpdorset.co.uk</li>
+                    <li>mention the sponsorship in the meeting announcement email on the mailing list</li>
+                    <li>tweet about the sponsorship from @phodorset at least once to announce it and once on the day as a reminder</li>
+                    <li>mention you at the start of the meeting, and make sure you have 30seconds or so during our announcements section to talk to the group</li>
+                </ul>
+            </p>
+
+            <p>We also have community sponsorship from <a href="https://www.jetbrains.com/">Jetbrains</a>, who provide free licenses to their awesome
+            software for our competitions, and from <a href="http://www.oreilly.com/">O'Reilly</a> books who provide a user group discount for our members</p>
+
+            <h2><a name="about-this">About this document</a></h2>
+
+            <p>This document was taken from the <a href="http://lrug.org/">LRUG.org</a> Readme, and modified for PHPDorset.  It is supposed to be a living document and as such contributions
+            are welcome via our <a href="https://github.com/PHPDorset/phpdorset.github.io">Github repository</a></p>
+
+            <h3><a name="license">License</a></h3>
+
+            <p>This text is licensed as <a href="http://creativecommons.org/about/cc0">CC0</a>; in summary, it is in the public domain and can be used and modified with or without attribution.</p>
```

Comment:
I considered that. I'm never sure of the minefield of licensing, and the CC0 was on the original document so I kept it in the PHPDorset version.  

I suppose if they are compatible (which from just looking seems so) then you're right and we could remove this to avoid confusion